### ISSUE TITLE
Try practica like real-user and report about your experience

### ISSUE THREAD

**Issue by goldbergyoni** [practicajs/practica#158]
**🎯 Goal:** - What can be improved in Practica in terms of developer experience? Develop some simple backend up (e.g, to-do application), generate skeleton code with practica and try to code your app. Note down issues and insights

**Comment by goldbergyoni** [comment]
@jovicon Wait 2 days! I'm deploying Fastify support (our default web app from now on), interested?

**Comment by jovicon** [comment]
> @jovicon Wait 2 days! I'm deploying Fastify support (our default web app from now on), interested?

Yes, I'm interested to help you and contribute with this project.
I'm a NestJS active learner and specialist, so I would help with nest too.
